Description: ENVIRONMENT section being filled, DESCRIPTION section being moved up, made a minor text formatting
Author: Konstantin Suchkov <ak.vokchus@gmail.com>

diff -Naurd ctorrent-1.3.4.dnh3.3.2~/debian/ctorrent.1 ctorrent-1.3.4.dnh3.3.2/debian/ctorrent.1
--- ctorrent-1.3.4.dnh3.3.2~/debian/ctorrent.1	2018-05-21 11:33:34.398587820 +0300
+++ ctorrent-1.3.4.dnh3.3.2/debian/ctorrent.1	2018-05-21 11:39:48.912831568 +0300
@@ -134,8 +134,12 @@
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
 .Vb 1
-\&  ctorrent \- [options] *.torrent
+ctorrent [OPTIONS] *.torrent
 .Ve
+.SH "DESCRIPTION"
+.IX Header "DESCRIPTION"
+CTorrent is a BitTorrent Client program written in C/\*(C+. Fast and
+small are CTorrent's two strengths.
 .SH "OPTIONS"
 .IX Header "OPTIONS"
 .IP "\fB\-a\fR" 4
@@ -209,19 +213,20 @@
 .IP "\fB\-z SLICE-SIZE\fR" 4
 .IX Item "-z SLICE-SIZE"
 Download slice/block size, unit \s-1KB\s0 (default 16, max 128)
-.SH "DESCRIPTION"
-.IX Header "DESCRIPTION"
-CTorrent is a BitTorrent Client program written in C/\*(C+. Fast and
-small are CTorrent's two strengths.
-.PP
-\&\s-1EXAMPLES\s0
-.PP
-.Vb 1
-\&    ctorrent \-s save.iso \-e 12 \-C 32 \-p 6881 iso.torrent
-.Ve
+.SH "EXAMPLES"
+.IX Header "EXAMPLES"
+ctorrent \-s save.iso \-e 12 \-C 32 \-p 6881 iso.torrent
+
+http_proxy=proxy1.example.com ctorrent John_Smith.torrent
 .SH "ENVIRONMENT"
 .IX Header "ENVIRONMENT"
-None.
+.IP "\fB\/http_proxy\fR" 4
+.IX Item "http_proxy"
+Use FQDN or IP address of a proxy server to redirect the client's
+queries. By default proxy server's port is 3128. Otherwise expli-
+citly denote a port number after a colon.
+( example: http_proxy=proxy2.example.com:12345 ).
+Note: It isn't implemented for CTCS.
 .SH "FILES"
 .IX Header "FILES"
 None.
